client/media/books.md

```diff
@@ -0,0 +1,13 @@
+# Books
+
+Here is a curated list of useful books to learn Coq.
+
+- [Coq'art](https://www.labri.fr/perso/casteran/CoqArt/) (Yves Bertot, Pierre Castéran) is the first book dedicated to the Coq Proof assistant and its underlying theory. Only the french version and the [sources and exercices](https://github.com/coq-community/coq-art) are free. The english version can purchased from the [Springer website](https://link.springer.com/book/10.1007/978-3-662-07964-5)
+- [Software Foundations](https://softwarefoundations.cis.upenn.edu/) (Benjamin Pierce et al., 2007, with regular updates), a series of Coq-based textbooks on logic, functional programming and foundations of programming languages , much acclaimed for being accessible to beginners, but rather oriented to computer scientists.
+- [Certified Programming with Dependent Types](http://adam.chlipala.net/cpdt/) (Adam Chlipala, 2008), a textbook about practical engineering with Coq, teaches advanced practical tricks and a very specific style of proof.
+- [Mathematical Components](https://math-comp.github.io/mcb/) (Assia Mahboubi and Enrico Tassi, 2018), a book that is more oriented towards mathematically inclined users, to dive into Coq with the SSReflect proof language, and the Mathematical Components library.
```

Comment:
For [our course](https://mpri-prfa.github.io/#references) we also mention:
- I. Sergey. [Programs and Proofs: Mechanizing Mathematics with Dependent Types](https://ilyasergey.net/pnp/). Lecture Notes.
- G. Smolka. [Modeling and Proving in Computational Type Theory Using the Coq Proof Assistant](https://www.ps.uni-saarland.de/~smolka/drafts/mpctt.pdf). Lecture Notes.